Userland/Applications/Welcome/WelcomeWidget.cpp

```diff
@@ -17,7 +17,6 @@
 #include <LibGUI/Process.h>
 #include <LibGfx/Font/BitmapFont.h>
 #include <LibGfx/Palette.h>
-#include <LibWebView/OutOfProcessWebView.h>
```

Comment:
commit message nit:
> Welcome: Remove unnecessary header 

should probably be:
`Welcome: Remove unnecessary include`